lib/concentrate/vehicle_position.ex

```diff
@@ -152,7 +154,8 @@ defmodule Concentrate.VehiclePosition do
               second.layover_departure_time,
               first.sources,
               first.layover_departure_time
-            )
+            ),
+          crowding: merge_crowding(first.crowding, second.crowding)
```

Comment:
Crowding data will all come from the same source and we'd never need to take half the object from one source and half from another, so I think `first_value(first.crowding, second.crowding)` would be fine here, rather than diving into the struct.